components/script/script_task.rs

```diff
@@ -915,7 +1030,7 @@ impl ScriptTask {
             MainThreadScriptMsg::Common(
                 CommonScriptMsg::FireTimer(TimerSource::FromWorker, _)) =>
                 panic!("Worker timeouts must not be sent to script task"),
-            MainThreadScriptMsg::Common(CommonScriptMsg::RunnableMsg(runnable)) =>
+            MainThreadScriptMsg::Common(CommonScriptMsg::RunnableMsg(_, runnable)) =>
                 runnable.handler(),
```

Comment:
They are being accounted for by [categorize_msg](https://github.com/servo/servo/pull/7341/files#diff-eeefd668773bcf179c9bdcf09d495b27R909). I suppose nested calls (i.e. from sync XHR) are not broken down further, but for that we'd need a different data structure…